homeassistant/components/ssdp/__init__.py

```diff
@@ -144,7 +144,7 @@ def __getitem__(self, name: str) -> Any:
         # Use a property if it is available, fallback to upnp data
         if hasattr(self, name):
             return getattr(self, name)
-        return self.upnp.get(name)
+        return self.upnp[name]
```

Comment:
This is to make sure it raises a KeyError as it would have done in the past.
Apparently Konnected relies on that:
https://github.com/home-assistant/core/blob/fb909eca8a643447f0a8355c0125a053a58ce77f/homeassistant/components/konnected/config_flow.py#L265-L266